### PR TITLE
fix(upgrade-node-core-signoff): upgrade ibm-cloud-sdk-core to v4

### PR DESCRIPTION
**Pull request by moisesatwork**
Signed-off-by: Moises Aaron Guerra Garcia <moises.guerra@ibm.com>

## PR summary

**Fixes:** <! -- link to issue -->

## PR Checklist
Please make sure that your PR fulfills the following requirements:
- [ ] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] New tests
- [ ] Build/CI related changes
- [ ] Documentation content changes
- [x] Other (please describe)

## Does this PR introduce a breaking change?
- [ ] Yes
- [ ] No

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,7 +3,6 @@ language: node_js
 sudo: false
 
 node_js:
-- 12
```
Comment (jkalandaibm) [inline .travis.yml:6]: Good job, figuring this out.. But why are you taking out version version 12 from the travis file:?

Comment (moisesatwork) [reply .travis.yml:6]: @jkalandaibm is 12 even needed? Based on Jest 29 upgrade documentation, it doesn't support node engine version 12: https://jestjs.io/blog/2022/08/25/jest-29

Comment (jkalandaibm) [reply .travis.yml:6]: You're right.. Then according to this:m https://github.com/IBM/platform-services-node-sdk/blob/main/.travis.yml#L5
If you could add 16 and 18 to it, that would be fantastic

**Review by kennburger — APPROVED** [review]

**Review by jkalandaibm — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (moisesatwork) [reply .travis.yml:6]: I will add 16 and 18

Comment (moisesatwork) [reply .travis.yml:6]: Yes, I will add it in a different PR

Comment (moisesatwork) [reply .travis.yml:6]: @jkalandaibm I created the second PR:
https://github.com/IBM/networking-node-sdk/pull/102
It seems the build for node_js 18 failed 😓

Comment (moisesatwork) [reply .travis.yml:6]: The error in the [node_js v18 CI run](https://app.travis-ci.com/github/IBM/networking-node-sdk/jobs/594834990) is:
```
node: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.27' not found (required by node)
```
Based on this `nvm` github issue that I found, it seems to be a problem related with the node and not nvm
https://github.com/nvm-sh/nvm/issues/2972

@jkalandaibm

**Pull request merged**